examples/tour/src/main.rs

```diff
@@ -699,19 +733,36 @@ impl<'a> Step {
     }
 }
 
-fn ferris<'a>(width: u16) -> Container<'a, StepMessage> {
+/// Passing fit=None defaults to ContentFit::Contain
+fn ferris<'a>(
+    trans: bool,
+    height: u16,
+    fit: Option<ContentFit>,
+) -> Container<'a, StepMessage> {
     Container::new(
         // This should go away once we unify resource loading on native
         // platforms
         if cfg!(target_arch = "wasm32") {
-            Image::new("tour/images/ferris.png")
+            if trans {
+                Image::new("tour/images/trans_ferris.png")
```

Comment:
I think we should keep the project decoupled from any symbolism related to gender and/or sex!

Maybe we could use the project logo instead?

Reply:
if i may, Ferris is [cannonically][1] [trans & nb][2], so any ferris is a trans ferris.  I'd be curious why you're concerned about explicitly including a trans flag tho?  that said, you're the maintainer, and if you really want the trans flag gone, ill take it out if you say the word.

[1]: https://twitter.com/whoisaldeka/status/1165147320964415489
[2]: https://www.rust-lang.org/learn/get-started#ferris